framework/scenario/tests/contract_without_macros.rs

```diff
@@ -286,7 +286,7 @@ mod sample_adder {
 
     impl<A> multiversx_sc::contract_base::CallableContract for ContractObj<A>
     where
-        A: multiversx_sc::api::VMApi,
+        A: multiversx_sc::api::VMApi + Send + Sync,
```

Comment:
Ideally, the contract shouldn't know about Send/Sync. Doesn't it work without this?

Reply:
Done by adding the constraint Send + Sync on traits VMApi and VMHooksApiBackend